Fail to start VMs with multiple disks

Attempting to use multiple disks, e.g., with `DISK_COUNT=3 make up` will fail to start the VMs with the following error message:
```
A customization command failed:

["storageattach", :id, "--storagectl", "SATAController", "--port", 1, "--device", 1, "--type", "hdd", "--medium", ".vagrant/master-disk-2.vdi"]

The following error was experienced:

#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["storageattach", "8f34c8d7-b3ca-4a35-8138-43ee27c93fa9", "--storagectl", "SATAController", "--port", "1", "--device", "1", "--type", "hdd", "--medium", ".vagrant/master-disk-2.vdi"]

Stderr: VBoxManage: error: The port and/or device parameter are out of range: port=1 (must be in range [0, 29]), device=1 (must be in range [0, 0])
VBoxManage: error: Details: code NS_ERROR_INVALID_ARG (0x80070057), component StorageControllerWrap, interface IStorageController, callee nsISupports
VBoxManage: error: Context: "AttachDevice(Bstr(pszCtl).raw(), port, device, DeviceType_HardDisk, pMedium2Mount)" at line 776 of file VBoxManageStorageController.cpp
>

Please fix this customization and try again.
make: *** [start-master] Error 1
```

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -111,7 +111,7 @@ Vagrant.configure('2') do |config|
                 unless File.exist?(".vagrant/master-disk-#{diskI}.vdi")
                     vb.customize ['createhd', '--filename', ".vagrant/master-disk-#{diskI}.vdi", '--variant', 'Standard', '--size', DISK_SIZE_GB * 1024]
                 end
-                vb.customize ['storageattach', :id, '--storagectl', 'SATAController', '--port', diskI - 1, '--device', diskI - 1, '--type', 'hdd', '--medium', ".vagrant/master-disk-#{diskI}.vdi"]
+                vb.customize ['storageattach', :id, '--storagectl', 'SATAController', '--port', diskI - 1, '--device', 0, '--type', 'hdd', '--medium', ".vagrant/master-disk-#{diskI}.vdi"]
             end
         end
         subconfig.vm.synced_folder 'data/master/', '/data', type: 'rsync',

--- a/Vagrantfile_nodes
+++ b/Vagrantfile_nodes
@@ -88,7 +88,7 @@ Vagrant.configure('2') do |config|
                 unless File.exist?(".vagrant/node#{i}-disk-#{diskI}.vdi")
                     vb.customize ['createhd', '--filename', ".vagrant/node#{i}-disk-#{diskI}.vdi", '--variant', 'Standard', '--size', DISK_SIZE_GB * 1024]
                 end
-                vb.customize ['storageattach', :id, '--storagectl', 'SATAController', '--port', diskI - 1, '--device', diskI - 1, '--type', 'hdd', '--medium', ".vagrant/node#{i}-disk-#{diskI}.vdi"]
+                vb.customize ['storageattach', :id, '--storagectl', 'SATAController', '--port', diskI - 1, '--device', 0, '--type', 'hdd', '--medium', ".vagrant/node#{i}-disk-#{diskI}.vdi"]
             end
         end
         subconfig.vm.synced_folder "data/node#{i}/", '/data', type: 'rsync',

